### DISCUSSION TITLE
Create Representation Window Error Session is not available for this project

### DISCUSSION THREAD

**Discussion by sena1771** [eclipse-sirius/sirius-desktop#409] [Q&A]
My goal is creating an eclipse modeling project. I choose viewpoint from the Viewpoints Selection window. Then, on the Create a New Representation window, selecting my diagram . On the console it gives me an error : "Session is not available for this project" when i pressed Next , then there is a empty screen and can not choose a semantic element for the new representation.

But this situation does not happen when the workspace is null , if the workspace is null then i can create my model project with representation and with whatever name i desired.

If on the workspace there is a project named "study" and if i try to create a new modeling project then I am taking the error "Session is not available" .

With deleting the "study" project and then try to create modeling project the error goes and i can correctly create modeling project with true representation.
This situation happens with spesific names for example "study" and texts starts with "c" letter . I do not know why it is happening and why with these spesific names i can not choose representation for my viewpoint.
Is there a way to solve this problem?

**Comment by pcdavid** [comment]
Sorry but from just your textualk description I'm not sure I understand the context/steps.
Can you provide some kind of screenshot, or better a short video of what you are doing?
